Main.html

```diff
@@ -7,12 +7,14 @@
 
```

Comment:
Move your link to css to the head.
Also you forgot about main **html** tag.
And you also need to set a **doctype**.

https://www.sitepoint.com/a-basic-html5-template/

Reply:
fixed